lib/child_process.js

```diff
@@ -650,8 +650,8 @@ function execFileSync(/* command, args, options */) {
 exports.execFileSync = execFileSync;
 
 
-function execSync(command /* , options */) {
-  var opts = normalizeExecArgs.apply(null, arguments);
+function execSync(command, options) {
+  var opts = normalizeExecArgs(command, options, null);
```

Comment:
I suggest to remove `null` as it's not required but by passing it through explicitly it somewhat seems like that.

Reply:
Last I heard, it was still better to pass the exact number of expected arguments. If that has changed, I'm happy to remove the optional parameter.